flow.tcl

```diff
@@ -177,6 +177,7 @@ proc run_non_interactive_mode {args} {
     set ANTENNACHECK_ENABLED [expr ![info exists flags_map(-no_antennacheck)] ]
 
     set steps [dict create \
+        "verilator_lint" "run_verilator" \
```

Comment:
Why not `run_verilator_step` and have a `RUN_VERILATOR` variable, to match some of the other steps?